src/dct/enum.lua

```diff
@@ -219,4 +219,25 @@ enum.event = {
 		--]]
 }
 
+enum.kickCode = {
+	["NOKICK"]  = 0,
+	["UNKNOWN"] = 1,
+	["SETUP"]   = 2,
+	["EMPTY"]   = 3,
+	["DEAD"]    = 4,
+	["LOADOUT"] = 5,
+	["MISSION"] = 6,
+}
+
+enum.kickReason = {
+	[enum.kickCode.NOKICK] =
+		"no kick requested, should never see this report bug",
+	[enum.kickCode.UNKNOWN] = "unknown reason",
+	[enum.kickCode.SETUP] = "slot setup, you joined too early",
```

Comment:
I'll test this soon, would like to rewrite some of these messages to be more "friendly" to a player though.

Reply:
If you have specific changes please suggest them. That is what a review is for. They are error messages "friendly" should not matter. What should matter in this case is the reason for the kick from the slot.